### DISCUSSION TITLE
点歌：GPT-4 Technical Report

### DISCUSSION THREAD

**Discussion by zmzhang** [mli/paper-reading#238] [General]
[GPT-4 Technical Report](https://cdn.openai.com/papers/gpt-4.pdf)

**Comment by zirui** [comment]
单看这篇论文(技术报告) ， 总结就是一句话:"我们训练了一个模型，效果更好", 
没有任何技术细节好讲的

**Reply by zmzhang** [reply]
GPT-4的多模态和32K上下文比较有意思。
> 单看这篇论文(技术报告) ， 总结就是一句话:"我们训练了一个模型，效果更好", 没有任何技术细节好讲的

**Reply by zirui** [reply]
> GPT-4的多模态和32K上下文比较有意思。
> 
> > 单看这篇论文(技术报告) ， 总结就是一句话:"我们训练了一个模型，效果更好", 没有任何技术细节好讲的

1. 多模态不如看看沐神之前的几个相关的视频，技术上应该没有什么新东西。
2. 32k是如何实现的是一个有意思的点， 但OpenAI的这篇98页的技术报告没有提供任何的技术细节。
总之，这篇技术报告完全是秀肌肉的PR文， 如果对以上两个点感兴趣，应该去看看别的论文或者等待以后更多相关的技术披露
